templates/sshd.ubuntu18.erb

```diff
@@ -0,0 +1,15 @@
+@include common-auth
```

Comment:
Is this from a stock installation without modification?

Reply:
Yes. there wasn't any big changes.. except for the reorganizing of selinux part.